x/stake/client/cli/query_sidechain.go

```diff
@@ -738,6 +739,56 @@ func GetCmdQuerySideAllValidatorsCount(cdc *codec.Codec) *cobra.Command {
 	return cmd
 }
 
+// GetCmdQueryCrossStakeRewardByBscAddress implements the cross stake reward query command.
+func GetCmdQueryCrossStakeRewardByBscAddress(cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "cross-stake-reward",
+		Short: "Query the cross stake reward balance by BSC address",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			bscAddress, err := sdk.NewSmartChainAddress(args[0])
+			if err != nil {
+				return err
+			}
+
+			cliCtx := context.NewCLIContext().WithCodec(cdc)
+			sideChainId, _, err := getSideChainConfig(cliCtx)
+			if err != nil {
+				return err
+			}
+
+			params := stake.QueryCrossStakeRewardParams{
+				BaseParams: stake.NewBaseParams(sideChainId),
+				BscAddress: bscAddress,
+			}
+
+			bz, err := json.Marshal(params)
+			if err != nil {
+				return err
+			}
+
+			response, err := cliCtx.QueryWithData("custom/stake/"+stake.QueryCrossStakeReward, bz)
+			if err != nil {
+				return err
+			}
+
+			switch viper.Get(cli.OutputFlag) {
+			case "text":
+				fmt.Println("The reward balance is:", string(response))
+			case "json":
+				//TODO
+				fmt.Println("The reward balance is:", string(response))
```

Comment:
Can we return a simple json string here?

Reply:
I simplify this part by remove the output format option